### DISCUSSION TITLE
Changing default Index in Elastalert rules generated by playbook

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#8467] [Unsupported Versions] [answered]
I want to change the default index in elastalert rules generated by playbook to something different. Currently it will always use *:so-* as the default index. Where to change or add other indices so that the elastalert rules generated will use different indices.

**Comment by defensivedepth** [comment, marked as answer]
This is not currently possible, but we are looking at a few different ways to allow this to be customized.

**Reply by ben-sec** [reply]
Has this been done in the meanwhile? I would love to create Sigma rules that query the elastalert index *:elastalert_status* instead of the default SO index,
